lib/patches/active_record/quoting.rb

```diff
@@ -0,0 +1,21 @@
+if ::ActiveRecord::VERSION::STRING >= "5.0.0"
+  module Globalize
+    module ConnectionAdapters
+      module Quoting
+
+        private
+
+        def _quote(value)
+          case value
+            when Array      then
+              "'#{quote_string(YAML.dump(value))}'"
+            else
+              super
+          end
+        end
+      end
+    end
+  end
+
+  ActiveRecord::ConnectionAdapters::AbstractAdapter.include Globalize::ConnectionAdapters::Quoting
```

Comment:
I think there is a problem on travis configuration. It seems that does'nt change the database adapter during testing and uses always sqlite. 
I expect no mistake when it uses postgres or mysql.